autogen/docs/dictionary.html.mustache

```diff
@@ -7811,438 +8602,487 @@ show subtract-headings 180 0
+          started). The potential resolution of the clock is milliseconds.
+          (Whether you get resolution that high in practice may vary from
+          system to system, depending on the capabilities of the underlying
+          Java Virtual Machine.)
+        </p>
+        <p>
+          See also <a href="#reset-timer">reset-timer</a>.
+        </p>
+        <p>
+          Note that the timer is different from the tick counter. The timer
+          measures elapsed real time in seconds; the tick counter measures
+          elapsed model time in ticks.
+        </p>
       </div>
-    <div class="dict_entry" id="to">
-      <h3>
-        <a>to</a>
-      </h3>
-      <h4>
-        <span class="prim_example">to <i>procedure-name</i></span>
-        <span class="prim_example">to <i>procedure-name</i> [<i>input1</i> ...]</span>
-      </h4>
-      <p>
-        Used to begin a command procedure.
-      <pre>
+      <div class="dict_entry" id="to">
+        <h3>
+          <a>to</a>
+        </h3>
+        <h4>
+          <span class="prim_example">to <i>procedure-name</i></span>
+          <span class="prim_example">to <i>procedure-name</i> [<i>input1</i> ...]</span>
+        </h4>
+        <p>
+          Used to begin a command procedure.
+        </p>
+        <pre>
 to setup
-  clear-all
-  crt 500
+clear-all
+crt 500
 end
 
 to circle [radius]
-  crt 100 [ fd radius ]
+crt 100 [ fd radius ]
 end
 </pre>
-    </div>
-    <div class="dict_entry" id="to-report">
-      <h3>
-        <a>to-report</a>
-      </h3>
-      <h4>
-        <span class="prim_example">to-report <i>procedure-name</i></span>
-        <span class="prim_example">to-report <i>procedure-name</i> [<i>input1</i> ...]</span>
-      </h4>
-      <p>
-        Used to begin a reporter procedure.
-      <p>
-        The body of the procedure should use <code>report</code> to report a
-        value for the procedure. See <a href="#report">report</a>.
-      <pre>
+      </div>
+      <div class="dict_entry" id="to-report">
+        <h3>
+          <a>to-report</a>
+        </h3>
+        <h4>
+          <span class="prim_example">to-report <i>procedure-name</i></span>
+          <span class="prim_example">to-report <i>procedure-name</i> [<i>input1</i> ...]</span>
+        </h4>
+        <p>
+          Used to begin a reporter procedure.
+        </p>
+        <p>
+          The body of the procedure should use <code>report</code> to report a
+          value for the procedure. See <a href="#report">report</a>.
+        </p>
+        <pre>
 to-report average [a b]
-  report (a + b) / 2
+report (a + b) / 2
 end
 
 to-report absolute-value [number]
-  ifelse number &gt;= 0
-    [ report number ]
-    [ report (- number) ]
+ifelse number &gt;= 0
+[ report number ]
+[ report (- number) ]
 end
 
 to-report first-turtle?
-  report who = 0  ;; reports true or false
+report who = 0  ;; reports true or false
 end
 </pre>
```

Comment:
This is better still!  But I'm seeing some `<pre>` section indentation is still lost - particular towards the end of the dictionary file - `to`, `to-report`, and a few more.